.circleci/config.yml

```diff
@@ -0,0 +1,135 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/2.0/configuration-reference/#jobs
+jobs:
+  build-linux:
+    # Specify the execution environment. You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
+    machine:
+      image: ubuntu-2004:current
+    resource_class: large
+    # Add steps to the job
+    # See: https://circleci.com/docs/2.0/configuration-reference/#steps
+    steps:
+      - checkout
+      - run:
+          name: "Set up build tools"
+          command: |
+            sudo apt-get update 
+            sudo apt-get install -y make clang llvm lld
+            sudo apt-get install -y \
+                  libgstreamer1.0-dev \
+                  libgstreamer-plugins-base1.0-dev \
+                  libgstreamer-plugins-good1.0-dev \
+                  libglib2.0-dev \
+                  libgl1-mesa-dev \
+                  libglu1-mesa-dev \
+                  libsm-dev \
+                  libx11-dev \
+                  libx11-xcb-dev \
+                  libexpat-dev \
+                  libxkbcommon-dev \
+                  libxcb1-dev \
+                  libxcb-glx0-dev \
+                  libxcb-icccm4-dev \
+                  libxcb-image0-dev \
+                  libxcb-keysyms1-dev \
+                  libxcb-randr0-dev \
+                  libxcb-render0-dev \
+                  libxcb-render-util0-dev \
+                  libxcb-shape0-dev \
+                  libxcb-shm0-dev \
+                  libxcb-sync-dev \
+                  libxcb-xfixes0-dev \
+                  libxcb-xinerama0-dev \
+                  libxcb-xkb-dev \
+                  libxcb-util-dev
+      - run:
+          name: "Build dependencies"
+          command: make -j 4 Dependencies MODE=release
+          no_output_timeout: 30m
+      - run:
+          name: "Build"
+          command: make -j 4 MODE=release
+          no_output_timeout: 30m
+      - store_artifacts:
+          path: ~/project/build/make/bin/linux-x64/Studio
+          destination: Studio
+
+  build-macos:
+    macos:
+      xcode: 13.3.1
+    steps:
+      - checkout
+      - run:
+          name: "Set up build tools"
+          command: |
+            brew install make
```

Comment:
This looks wrong to me... I have never used or needed `brew` to build Studio on OSX and it is not mentioned in our build notes that way. The requirements for OSX are XCode and the XCode Command Line Tools. Afterwards you should have all necessary tools (like make, clang, etc.) on your system.